programming-basics/fibonacci/main.js

```diff
@@ -0,0 +1,6 @@
+const fib = (valor) => {
+  // insira seu código aqui e retorne a resposta
+  return //resposta
+}
+
+export default fib
```

Comment:
It's a good practice to add an empty line on the EOF (end of file).

Check this link for more information: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file

Reply:
k, also setting auto-linting